app/celery/tasks.py

```diff
@@ -254,6 +266,15 @@ def save_email(self,
     service = dao_fetch_service_by_id(service_id)
     template = dao_get_template_by_id(notification['template'], version=notification['template_version'])
 
+    personalisation = notification.get('personalisation')
+    try:
+        validate_template(template.id, personalisation, service, template.template_type)
+    except BadRequestError as e:
+        current_app.logger.debug(
+            "SMS {} failed: {}".format(notification_id, e.message)
```

Comment:
This says the word SMS in but the function it's running in is called save_email. Assume this will be changed later.